tests/CMakeLists.txt

```diff
@@ -26,7 +26,10 @@ else()
     set(LIBRARIES socket)
 endif()
 
-if (WITH_SYSTEMD OR WITH_SYSTEMD_SOCKET_ACTIVATION OR WITH_SYSTEMD_WATCHDOG)
+if (WITH_SYSTEMD
+        OR WITH_SYSTEMD_SOCKET_ACTIVATION
+        OR WITH_SYSTEMD_WATCHDOG
+        OR WITH_SYSTEMD_JOURNAL)
```

Comment:
Why this change relates to the change of trace limit?

Reply:
I wanted to make sure all tests are getting enabled properly when testing this as the condition previously did not work correctly. I can move it to another PR though

Reply:
Hello,
It's okay splitting it to a separate commit in this PR.